### PR TITLE
100-Correct prices

### PR DESCRIPTION
**Pull request by matt-paul**
## Pre-flight check-list

for issue #100 

Before raising a pull request

* [ ] Documentation
* [x] Unit tests

## Pre-merge check-list

* [x] Link to Trello ticket/GitHub issue (If applicable) #100
* [ ] Any risky areas identified
* [ ] Test plan provided
* [ ] Tester approved

## Tester check-list

* [ ] Tested on multiple devices / OS versions (Test on the physical device(s) you have access to, otherwise use BrowserStack):

  * [ ] Galaxy S8 (Android 7.0/8.0)
  * [ ] Galaxy S7 (Android 6.0)
  * [ ] iPhone X (iOS 11.x)
  * [ ] iPhone 8 (11.x)
  * [ ] iPhone 7 (iOS 10.x)

  Optional:

  * [ ] Google Pixel 2
  * [ ] Galaxy S8 Plus
  * [ ] Galaxy S7 Edge
  * [ ] Galaxy S6
  * [ ] iPhone 7+
  * [ ] iPhone 6

Accessibility Testing (If applicable)

* [ ] Text-to-Voice (Android: Voice Assistant / iOS: Speak Selection)
* [ ] Large Text (=<200%)
* [ ] Landscape Screen orientation

Weak connection testing (If applicable)

* [ ] Airplane mode
* [ ] 2G/3G Network Settings

### CODE REVIEW

**Comment by zoeabryant** [comment]
need to think about tomorrow:

- what should happen if the user only selects one price (currently : https://github.com/redbadger/pride-london-app/pull/257/files#diff-592ac38477aabfc26e974e01cc20dd9dR278)

- what happens if the user sets the same price for both fields....

**Comment by matt-paul** [comment]
BOTH ZERO:
if priceLow: 0
if priceHigh: 0
eventTile should say: Free
event price overview should say: Free

RANGE FROM FREE TO PRICE:
if priceLow: 0
if priceHigh: 10
eventTile should say: from £0
event price overview should say: £0 - £10

RANGE OF PRICES:
if priceLow: 5
if priceHigh: 10
eventTile should say: from £5
event price overview should say: £5 - £10

SAME PRICE:
if priceLow: 10
if priceHigh: 10
eventTile should say: £10
event price overview should say: £10

INCORRECT PRICE RANGE SET: (we will flip and correct these)
if priceLow: 10
if priceHigh: 0
eventTile should say: from £0
event price overview should say: £0 - £10

File: src/data/formatters.test.js [hunk 1]
```diff
@@ -3,8 +3,8 @@ import {
   formatDateRange,
   formatTime,
   formatPrice,
-  formattedEventPrice,
-  formattedEventPriceRange,
+  formattedShortEventPrice,
+  formattedLongEventPrice,
```
Comment (RGBboy) [inline src/data/formatters.test.js]: I would rename these to follow the same tense as the other format functions. I.e. `formatShortEventPrice` and `formatLongEventPrice`.

File: src/data/formatters.test.js [hunk 2]
```diff
@@ -55,38 +55,51 @@ describe("formatPrice", () => {
...
 
-describe("formattedEventPriceRange", () => {
+describe("formattedLongEventPrice", () => {
   it("returns free for free events", () => {
-    expect(formattedEventPriceRange(true, 1, 2)).toEqual("Free");
+    expect(formattedLongEventPrice(0, 0)).toEqual("Free");
+  });
+
+  it("returns the range of prices when zero and a higher price are given", () => {
+    expect(formattedLongEventPrice(0, 2)).toEqual("£0 - £2");
```
Comment (RGBboy) [inline src/data/formatters.test.js]: The separator here should be an en-dash `–`, not a hyphen `-`.

Comment (matt-paul) [reply src/data/formatters.test.js]: 👍

File: src/data/formatters.js [hunk 3]
```diff
@@ -25,13 +26,12 @@ export const formatPrice = (price: number) => {
...
 
-export const formattedEventPrice = (
-  isFree: boolean,
+export const formatShortEventPrice = (
   eventPriceLow: number,
   eventPriceHigh: number
 ) => {
   let displayPrice;
-  if (isFree) {
+  if (isFree(eventPriceLow, eventPriceHigh)) {
```
Comment (RGBboy) [inline src/data/formatters.js]: Not going to block, but you can refactor function to return early for each case instead of using a `let` statement.

**Pull request merged**